fingerprint/capture/src/main/java/com/simprints/fingerprint/capture/screen/FingerprintCaptureFragment.kt

```diff
@@ -104,9 +111,26 @@ internal class FingerprintCaptureFragment : Fragment(R.layout.fragment_fingerpri
             args.params.fingerprintSDK,
         )
 
+        scanningStatusTracker.scanCompleted.observe(viewLifecycleOwner, LiveDataEventObserver {
+            if (isAudioEnabled())
+                playBeep()
+        })
         initUI()
     }
 
+    private fun playBeep() {
+        val mediaPlayer = MediaPlayer.create(context, R.raw.beep)
```

Comment:
Can the `MediaPlayer` instance be created on the Fragment properties level so that we don't have to re-initialize it every time? If memory serves me right, the `MediaPlayer` creation might be resource-intensive due to codec and decoders initialization etc.

Reply:
If you decide that creating a media player instance makes sense, then please also make sure that it is re-created correctly once the device is rotated

Reply:
I would also suggest moving it together with the `PreferenceManager` to some kind of injectable (ideally singleton) manager class.